docs/maps/maps-aggregations.asciidoc

```diff
@@ -2,13 +2,13 @@
 [[maps-aggregations]]
 == Plot big data without plotting too much data
 
-The Maps application uses {ref}/search-aggregations.html[aggregations] to plot large data sets without overwhemling your network or your browser.
+Use {ref}/search-aggregations.html[aggregations] to plot large data sets without overwhemling your network or your browser.
 
 Aggregations group your documents into buckets and calculate metrics for each bucket.
 Your documents stay in Elasticsearch and only the metrics for each group are returned to your computer.
 
 
-[float]
```

Comment:
Can we put the float tags back in?  There are now some really short pages.

Reply:
I had to take the float out to get the titles working under `Terms join`. If there is another way to get a deeper set of titles with float that would be great, but I could not get them to work without putting `Terms join` on its own page